### DISCUSSION TITLE
How to edit / upload config file

### DISCUSSION THREAD

**Discussion by kragil** [louislam/dockge#517] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

For a lot of containers, you have to create or upload config files. How would I go about doing that with Dockge only?

### 🐻 Dockge Version

current

### 💻 Operating System and Arch

Debian

### 🌐 Browser

Firefox

**Comment by rowanmoul** [comment]
I came here to ask the same thing.
~~I think the answer is that you don't, unless you want to get hacky and run a shell command in the container on startup to build the config file from env vars and save it wherever the application will be looking for it.~~

Apparently you can define config files fully within the compose file now:
https://docs.docker.com/reference/compose-file/configs
https://docs.docker.com/reference/compose-file/services/#configs

The best part of this is that you can use .env values in the config files too.
